packages/expo-asset/src/Asset.js

```diff
@@ -59,6 +59,22 @@ const resolveIfRelative = uri => {
   }
 };
 
+function isImageType(type) {
+  return type.match(/(jpeg|jpg|gif|png|bmp)$/) != null;
```

Comment:
```suggestion
  return /^(jpeg|jpg|gif|png|bmp)$/i.test(type);
```

This returns a boolean instead of allocating the matched substring and we should be lenient about case sensitivity here. Also better to look for an exact match unless we want to handle APNG files, for example.